pages/implementation/inspector/sdk/go.mdx

```diff
@@ -0,0 +1,118 @@
+---
+layout: 'guide'
+title: 'Avo Inspector Go SDK'
+---
+
+import Link from '../../../../components/Link';
+
+## <a name="quick-start"></a>Quick Start Guide
+
+Find the Quick Start Guide in our <a target="_blank" rel="noopener noreferrer" href="https://github.com/avohq/go-avo-inspector/tree/develop">GitHub repo.</a>
+
+## <a name="installation"></a>Installation
+
+Our library is distributed as a go module:
+
+```go
+go get github.com/avohq/go-avo-inspector
+```
+
+Use the latest GitHub release tag to get the latest version of the library.
+
+## <a name="import"></a>Import
+
+```go
+import (
+	avoinspector "github.com/avohq/go-avo-inspector"
+)
+```
+
+## <a name="initialization"></a>Initialization
+
+Obtain the API key in Inspector tab in your <a target="_blank" rel="noopener noreferrer" href="https://www.avo.app/welcome">Avo.app</a> workspace.
+
+You will need to create an instance of `AvoInspector`.
+
+```go
+func NewAvoInspector(apiKey string, env AvoInspectorEnv, appVersion string, appName string) (*AvoInspector, error)
```

Comment:
Should we break this into multiple lines to avoid it cutting off and require horizontal scrolling?